lib/pub_sub/message.rb

```diff
@@ -1,7 +1,7 @@
 module PubSub
   class Message
     def initialize(payload)
-      @payload = JSON.parse(payload)
+      @payload = JSON.parse(JSON.parse(payload)['Message'])
```

Comment:
[These](https://gist.github.com/adamcohen/9b3a495a49dd210c8014) are all the current subscriptions with RawmessageDelivery set to false.  The interesting ones are:

     TopicArn: 'arn:aws:sns:us-east-1:717362228064:category-service-uat',
     Endpoint: 'arn:aws:sqs:us-east-1:717362228064:store-service-uat',

     TopicArn: 'arn:aws:sns:us-east-1:717362228064:store-service-production',
     Endpoint: 'arn:aws:sqs:us-east-1:717362228064:event-service-production',

     TopicArn: 'arn:aws:sns:us-east-1:717362228064:store-service-uat',
     Endpoint: 'arn:aws:sqs:us-east-1:717362228064:event-service-uat',

     TopicArn: 'arn:aws:sns:us-east-1:717362228064:centre-service-production',
     Endpoint: 'arn:aws:sqs:us-east-1:717362228064:collection-service-production',

     TopicArn: 'arn:aws:sns:us-east-1:717362228064:centre-service-uat',
     Endpoint: 'arn:aws:sqs:us-east-1:717362228064:collection-service-uat',

     TopicArn: 'arn:aws:sns:us-east-1:717362228064:category-service-production',
     Endpoint: 'arn:aws:sqs:us-east-1:717362228064:store-service-production',